### ISSUE TITLE
Hover over for details

### ISSUE THREAD

**Issue by viktak** [alexandrainst/alexandra-trackmap-panel#21]
I was wondering if it is possible to configure the map in a way that when the mouse hovers over a track point, it shows the details (or at least the time stamp) of that point, like any other graph in Grafana does, i.e.:

![grafana-hover-over-example](https://user-images.githubusercontent.com/12509879/90398261-ca808c00-e0a1-11ea-9e60-6b12729e41e9.png)

If it is not possible, please take this as a feature request :)

Thank you!

**Comment by viktak** [comment]
While at it :) can I also ask for a possibility of a small (custom) icon/image to be shown at the last point?

Imagine a live tracklog, where one can see, say, the last 24 hours, and the current position. At the moment it is not easy to see / not obvious which one the current position.

Thank you!!

**Comment by xkilian** [comment]
This is an especially important feature for any vehicule tracking that crosses back on its original path. (ex. car, bus, taxi, ferry type circuits)

**Comment by bijwaard** [comment]
At the moment I can see the lat/long coordinates in the tooltip, great! So there is already something showing up while hoovering;-)

Would it is possible to add a label to the tooltip. I now changed the icons to have an embedded label but that only works for two icons... It would be great if this label/id can be dynamically set from the metrics, e.g. by using the identity_id metric alias below.
![image](https://user-images.githubusercontent.com/6523124/98688478-2ab65f80-236b-11eb-9f0a-782b6fda0277.png)

**Comment by j-or** [comment]
@viktak @bijwaard 

For highlighting the last position, check out the _Use secondary icon for last marker_ option that was added recently.

For labels/tooltips, try #40 that I just pushed. It adds a mouseover tooltip/label to each marker using the _tooltip_ field from the query. It also has the option to _Always show tooltips_  if you want the tooltips to stay open, instead of only showing them on mouseover.
